vignettes/ectd/tlf-kmplot.R

```diff
@@ -64,7 +64,7 @@ ggplot2::theme_set(theme_bw())
 
 pdf.options(reset = TRUE, onefile = FALSE)
 
-pdf(file.path(path$output, "tlf-kmplot.pdf"))
+pdf(file.path(path$output, "tlf-kmplot-pilot3.pdf"))
```

Comment:
Hi, @robertdevine .  When I get to this part of the code, I get an error that shows :  
```
> pdf(file.path(path$output, "tlf-kmplot-pilot3.pdf"))
Error in file.path(path$output, "tlf-kmplot-pilot3.pdf") : 
  object 'path' not found
```

Did you get this error too?

Reply:
Thanks @laxamanaj. No, I do not. In fact no error results when running the Issue #32 PR branch code in my posit Cloud sandbox. The tlf-kmplot-pilot3.pdf is generated with correct {timestamp} in the footer. 

![image](https://user-images.githubusercontent.com/8195239/228385942-f85257cd-6bb7-4b9b-b5df-802432146692.png)

![image](https://user-images.githubusercontent.com/8195239/228386459-7e475ebb-7f9e-45ec-9cf2-d54bfcce71a3.png)

Reply:
@laxamanaj, in our meeting discussion on this issue we only talked about changes to the .R source, 'git mv' to change the name of the tracked km plot pdf file should address the error you are experiencing - then include the newly-named pdf in the PR. The ADRG note at the top of the file remains applicable for the ADRG Appendix in Pilot3. Thanks again.

Reply:
Also @laxamanaj , worth noting for Pilot3, the tlf-kmplot.Rmd does not append the '-pilot3' to the pdf name. Might be best for consistency to do so and track accordingly in the repository. Thanks.

Reply:
Thanks, @robertdevine .  I'm only running the script in `/cloud/project/vignettes/ectd/tlf-kmplot.R`, not the. RMD.

In my error it says I'm having issue with the `object 'path' not found`.  I see at the top of the `tlk-kmplot.R` script there are commented out lines for `path` and `path$output`.  Did you run these lines to set `path` first before running the rest of the code?

Reply:
It works, if I update the `path` in line 7, but I didn't see it in the code you pushed to this remote branch.  Can you re-push your current `/cloud/project/vignettes/ectd/tlf-kmplot.R` again so that I can re-pull? Thanks.

Reply:
Thank you, @laxamanaj. Should be good to re-pull and run from your posit Cloud sandbox with existing environment configuration and project package cache.

Reply:
Completed!  Thanks @robertdevine for your work on this.